config-connector/solutions/iam/kpt/pubsub-topic/README.md

```diff
@@ -0,0 +1,54 @@
+Pubsub Topic
+==================================================
+
+# NAME
+
+  pubsub-topic
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to grant a role to a particular member for a PubSub topic.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  BASE=https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+  kpt pkg get $BASE/config-connector/solutions/iam/kpt/pubsub-topic pubsub-topic
+  ```
+
+# REQUIREMENTS
+
+  A working cluster with Config Connector installed.
+  A GCP project with PubSub service enabled.
```

Comment:
It might flow better if you tie these thoughts together:
A working cluster with Config Connector installed, managing a GCP project... or something like that.

Reply:
Made the change.